services/harmony/app/backends/workflow-orchestration/work-item-updates.ts

```diff
@@ -564,9 +564,13 @@ export async function preprocessWorkItem(
     durationMs = new Date().getTime() - resultStartTime;
     logger.debug('timing.HWIUWJI.getResultItemSize.end', { durationMs });
   } catch (e) {
-    errorMessage = 'Could not get result item file size';
+    errorMessage = 'Could not get result item file size from output STAC';
```

Comment:
Couldn't come up with a better error message.

Reply:
I think maybe something like `Service ${service} failed to produce usable output` might work better for end users.

Reply:
Agree the message with service name is better, but the `service` is not available in this function and I am not sure if it is worth it to somehow get the service name in this function for logging only.

Reply:
Is there a way we can have different messages for different issues? For this particular one I think I'd say something like "Cannot parse STAC catalog output from service." and expect it would get thrown from trying to read the catalog and not a call to get the file size.

Reply:
Done.